### ISSUE TITLE
README

### ISSUE THREAD

**Issue by zya** [sofacoustics/API_Cpp#1]
An example use of the API would be beneficial to the new users.

**Comment by tcarpent** [comment]
you can find rough examples in the following files :

API_Cpp/libsofa/src/sofaexamples.cpp
API_Cpp/libsofa/src/sofainfo.cpp
API_Cpp/libsofa/src/sofamisc.cpp

**Comment by hagenw** [comment]
That's true, but I agree that it would be nice to add a README with some hints on how to use the API and the link to the three files for how get started.

**Comment by tcarpent** [comment]
Feel free to add it.

**Comment by hagenw** [comment]
Unfortunately, I have no clue at all about the C++ API ;)
